.gitignore

```diff
@@ -8,3 +8,7 @@ node_modules
 /cypress/videos
 /prisma/data.db
 /prisma/data.db-journal
+
+.pnpm-store/
+XL-Dockerfile
+docker-compose.yml
```

Comment:
```suggestion
```
* i think new version of pnpm has the store inside node_modules, try to update your pnpm. in the first place is this project even using pnpm?
* you don't have to ignore `XL-Dockerfile`, just don't include it in the commit
* you don't have to ignore `docker-compose.yml`, just don't include it in the commit

Reply:
i added it to the gitignore cause i cannot get it to start via dockerfile. :D

Reply:
gotta fix and and remove them from the `.gitignore`